### ISSUE TITLE
test suite fails if built without smartmontools support

### ISSUE THREAD

**Issue by mbiebl** [storaged-project/libblockdev#1061]
Version: 3.2.0

In Debian I decided to build the libblockdev package without smartmontools support (for now).

Still, when running the test suite, it seems to call `smartmontools_test.SmartmontoolsTest` which then fails
```
2892s ======================================================================
2892s ERROR: setUpClass (smartmontools_test.SmartmontoolsTest)
2892s ----------------------------------------------------------------------
2892s Traceback (most recent call last):
2892s   File "/tmp/autopkgtest.Ua8Dvc/build.ONh/src/tests/smartmontools_test.py", line 32, in setUpClass
2892s     BlockDev.reinit([cls.ps, cls.ps2], True, None)
2892s   File "/tmp/autopkgtest.Ua8Dvc/build.ONh/src/src/python/gi/overrides/BlockDev.py", line 164, in reinit
2892s     return _reinit(require_plugins, reload, log_func)
2892s            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2892s gi.repository.GLib.GError: g-bd-init-error-quark: Failed to load plugins (1)
2892s 
2892s ----------------------------------------------------------------------
2892s Ran 506 tests in 2808.874s
```

Full test log can be found at https://ci.debian.net/packages/libb/libblockdev/testing/amd64/51608847/

**Comment by tbzatek** [comment]
I believe this is another manifestation of #943

**Comment by vojtechtrefny** [comment]
Yes, that's the same issue. I plan to fix #943 before the next release, hopefully soon.
